.github/workflows/csharp.yml

```diff
@@ -6,41 +6,21 @@ on:
     branches: [master]
   pull_request:
     branches: [master]
+    
+env:
+  RELEASE_BRANCH: "3.3.x"
 
 jobs:
-  lint_code_base:
```

Comment:
Are we removing linting?  I know it was failing but...

Reply:
Sorry that was a mistake. Fixed it